sql/sys_vars.cc

```diff
@@ -1609,7 +1609,7 @@ static Sys_var_bool Sys_binlog_rows_query(
 
 static Sys_var_bool Sys_binlog_ddl_query(
     "binlog_ddl_skip_rewrite",
-    "Logs the input query  into the binary log without the server rewrite."
+    "Without server rewrite, DDL queries will be logged with comments."
     "This action is useful when the server-side DDL query strips comments.",
```

Comment:
Since we mention about the comments, we can remove this line "This action is useful.." Sorry, I wasn't clear about this in last comment.